src/apprt/gtk-ng/class/application.zig

```diff
@@ -778,9 +776,129 @@ pub const Application = extern struct {
                 \\.window headerbar {{
                 \\  font-family: "{[font_family]s}";
                 \\}}
+                \\
             , .{ .font_family = font_family });
         }
 
+        try loadRuntimeCss414(config, &writer);
+        try loadRuntimeCss416(config, &writer);
+
+        // ensure that we have a sentinel
+        try writer.writeByte('\x00');
+
+        const data = buf.items[0 .. buf.items.len - 1 :0];
+
+        log.debug("runtime CSS is {d} bytes", .{data.len + 1});
+
+        // Clears any previously loaded CSS from this provider
+        loadCssProviderFromData(
+            self.private().css_provider,
+            data,
+        );
+    }
+
+    /// Load runtime CSS for older than GTK 4.16
+    fn loadRuntimeCss414(
```

Comment:
```suggestion
    fn loadRuntimeCss4_14(
```

Reply:
Zig functions don't generally use `_` AFAICS so I left these as-is.